frontend/src/App.tsx

```diff
@@ -16,6 +17,7 @@ function App (): JSX.Element {
         <Route path="/tempLandingPage" element={<TempLandingPage />} />
         <Route path="/contactPage" element={<Contacts />} />
         <Route path="/planForm" element={<PlanForm />} />
+        <Route path="/planPage/:id" element={<Plan />} />
```

Comment:
do you want the url in the browser for the user to read `wewrap.com/planPage`? I think it normally would just be `wewrap.com/plans` or `wewrap.com/plan/:id`

Reply:
ok i switched the route to `/plan/:id`